modules/microshift-install-bootc-get-published-image.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// microshift_install_bootc/microshift-install-rhel-bootc-image.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-install-bootc-get-published-image_{context}"]
+= Getting the published bootc image for {microshift-short}
+
+Use the following procedure to find and use the {microshift-short} container images to install {op-system-base-full}.
+
+.Prerequisites
+
+* You have an x86_64 or AArch64 platform.
```

Comment:
```suggestion
* You have an x86_64 or aarch64 platform.
```

Reply:
The caps style is preferred by docs in this case, see https://redhat-documentation.github.io/supplementary-style-guide/#aarch64

Reply:
It does not say this is preferred. I think both are good and I'd prefer to see `aarch64` for consistency  with the `uname -m` command output.

Reply:
It is preferred in the MicroShift docs; we have used this style since 4.12 in plain text. It does not need to match a command.